### ISSUE TITLE
[10.0.5 RC3] Problems getting apps from Marketplace

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/core#30126]
**Steps:**

0. mv /var/www/owncloud /var/www/owncloud.old
1. extract *.tar ==> owncloud
2. replace config.php with my old one from owncloud.old
3. move data directory from owncloud.old
4. occ upgrade

**Upgrade didn't succeed:**

```
root@owncloud:/var/www/owncloud# sudo -u www-data php ./occ upgrade
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
2018-01-12T18:19:50+00:00 Set log level to debug
2018-01-12T18:19:50+00:00 Repair step: Repair MySQL database engine
2018-01-12T18:19:50+00:00 Repair step: Repair MySQL collation
2018-01-12T18:19:50+00:00 Repair info: All tables already have the correct collation -> nothing to do
2018-01-12T18:19:50+00:00 Repair step: Repair SQLite autoincrement
2018-01-12T18:19:50+00:00 Repair step: Repair duplicate entries in oc_lucene_status
2018-01-12T18:19:50+00:00 Repair info: lucene_status table does not exist -> nothing to do
2018-01-12T18:19:50+00:00 Repair step: Upgrade app code from the marketplace
2018-01-12T18:19:50+00:00 Repair info: Using market to update existing apps
2018-01-12T18:19:50+00:00 Repair info: Attempting to update the following missing apps from market: files_pdfviewer, files_texteditor, gallery, oauth2, xmas
2018-01-12T18:19:50+00:00 Repair info: Fetching app from market: files_pdfviewer
2018-01-12T18:19:52+00:00 
2018-01-12T18:19:52+00:00                                                                                                                                                                                                     
2018-01-12T18:19:52+00:00   [Symfony\Component\Debug\Exception\FatalThrowableError]                                                                                                                                           
2018-01-12T18:19:52+00:00   Type error: Argument 2 passed to OC_App::executeRepairSteps() must be of the type array, null given, called in /var/www/owncloud/lib/private/Installer.php on line 154  
2018-01-12T18:19:52+00:00                                                                                                                                                                                                     

2018-01-12T18:19:52+00:00 Exception trace:
2018-01-12T18:19:52+00:00  () at /var/www/owncloud/lib/private/legacy/app.php:1064
2018-01-12T18:19:52+00:00  OC_App::executeRepairSteps() at /var/www/owncloud/lib/private/Installer.php:154
2018-01-12T18:19:52+00:00  OC\Installer::installApp() at /var/www/owncloud/lib/private/App/AppManager.php:436
2018-01-12T18:19:52+00:00  OC\App\AppManager->installApp() at /var/www/owncloud/apps/market/lib/MarketService.php:131
2018-01-12T18:19:52+00:00  OCA\Market\MarketService->installPackage() at /var/www/owncloud/apps/market/lib/MarketService.php:120
2018-01-12T18:19:52+00:00  OCA\Market\MarketService->installApp() at /var/www/owncloud/apps/market/lib/Listener.php:46
2018-01-12T18:19:52+00:00  OCA\Market\Listener->reinstallAppStoreApp() at /var/www/owncloud/apps/market/lib/Application.php:55
2018-01-12T18:19:52+00:00  OCA\Market\Application->OCA\Market\{closure}() at n/a:n/a
2018-01-12T18:19:52+00:00  call_user_func() at /var/www/owncloud/lib/composer/symfony/event-dispatcher/EventDispatcher.php:174
2018-01-12T18:19:52+00:00  Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() at /var/www/owncloud/lib/composer/symfony/event-dispatcher/EventDispatcher.php:43
2018-01-12T18:19:52+00:00  Symfony\Component\EventDispatcher\EventDispatcher->dispatch() at /var/www/owncloud/lib/private/Repair/Apps.php:222
2018-01-12T18:19:52+00:00  OC\Repair\Apps->getAppsFromMarket() at /var/www/owncloud/lib/private/Repair/Apps.php:158
2018-01-12T18:19:52+00:00  OC\Repair\Apps->run() at /var/www/owncloud/lib/private/Repair.php:89
2018-01-12T18:19:52+00:00  OC\Repair->run() at /var/www/owncloud/lib/private/Updater.php:221
2018-01-12T18:19:52+00:00  OC\Updater->doUpgrade() at /var/www/owncloud/lib/private/Updater.php:108
2018-01-12T18:19:52+00:00  OC\Updater->upgrade() at /var/www/owncloud/core/Command/Upgrade.php:254
2018-01-12T18:19:52+00:00  OC\Core\Command\Upgrade->execute() at /var/www/owncloud/lib/composer/symfony/console/Command/Command.php:262
2018-01-12T18:19:52+00:00  Symfony\Component\Console\Command\Command->run() at /var/www/owncloud/lib/composer/symfony/console/Application.php:826
2018-01-12T18:19:52+00:00  Symfony\Component\Console\Application->doRunCommand() at /var/www/owncloud/lib/composer/symfony/console/Application.php:189
2018-01-12T18:19:52+00:00  Symfony\Component\Console\Application->doRun() at /var/www/owncloud/lib/composer/symfony/console/Application.php:120
2018-01-12T18:19:52+00:00  Symfony\Component\Console\Application->run() at /var/www/owncloud/lib/private/Console/Application.php:161
2018-01-12T18:19:52+00:00  OC\Console\Application->run() at /var/www/owncloud/console.php:106
2018-01-12T18:19:52+00:00  require_once() at /var/www/owncloud/occ:11
2018-01-12T18:19:52+00:00 
2018-01-12T18:19:52+00:00 upgrade [--no-app-disable]
2018-01-12T18:19:52+00:00 
```

**I tried it again, and output was slightly different:**

```
root@owncloud:/var/www/owncloud# sudo -u www-data php ./occ upgrade
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
2018-01-12T18:20:08+00:00 Set log level to debug
2018-01-12T18:20:08+00:00 Repair step: Repair MySQL database engine
2018-01-12T18:20:08+00:00 Repair step: Repair MySQL collation
2018-01-12T18:20:08+00:00 Repair info: All tables already have the correct collation -> nothing to do
2018-01-12T18:20:08+00:00 Repair step: Repair SQLite autoincrement
2018-01-12T18:20:08+00:00 Repair step: Repair duplicate entries in oc_lucene_status
2018-01-12T18:20:08+00:00 Repair info: lucene_status table does not exist -> nothing to do
2018-01-12T18:20:08+00:00 Repair step: Upgrade app code from the marketplace
2018-01-12T18:20:08+00:00 Repair info: Using market to update existing apps
2018-01-12T18:20:08+00:00 Repair info: Attempting to update the following missing apps from market: files_texteditor, gallery, oauth2, xmas
2018-01-12T18:20:08+00:00 Repair info: Fetching app from market: files_texteditor
2018-01-12T18:20:10+00:00 
2018-01-12T18:20:10+00:00                                                                                                                                                                                                     
2018-01-12T18:20:10+00:00   [Symfony\Component\Debug\Exception\FatalThrowableError]                                                                                                                                           
2018-01-12T18:20:10+00:00   Type error: Argument 2 passed to OC_App::executeRepairSteps() must be of the type array, null given, called in /var/www/owncloud/lib/private/Installer.php on line 154  
2018-01-12T18:20:10+00:00                                                                                                                                                                                                     

2018-01-12T18:20:10+00:00 Exception trace:
2018-01-12T18:20:10+00:00  () at /var/www/owncloud/lib/private/legacy/app.php:1064
2018-01-12T18:20:10+00:00  OC_App::executeRepairSteps() at /var/www/owncloud/lib/private/Installer.php:154
2018-01-12T18:20:10+00:00  OC\Installer::installApp() at /var/www/owncloud/lib/private/App/AppManager.php:436
2018-01-12T18:20:10+00:00  OC\App\AppManager->installApp() at /var/www/owncloud/apps/market/lib/MarketService.php:131
2018-01-12T18:20:10+00:00  OCA\Market\MarketService->installPackage() at /var/www/owncloud/apps/market/lib/MarketService.php:120
2018-01-12T18:20:10+00:00  OCA\Market\MarketService->installApp() at /var/www/owncloud/apps/market/lib/Listener.php:46
2018-01-12T18:20:10+00:00  OCA\Market\Listener->reinstallAppStoreApp() at /var/www/owncloud/apps/market/lib/Application.php:55
2018-01-12T18:20:10+00:00  OCA\Market\Application->OCA\Market\{closure}() at n/a:n/a
2018-01-12T18:20:10+00:00  call_user_func() at /var/www/owncloud/lib/composer/symfony/event-dispatcher/EventDispatcher.php:174
2018-01-12T18:20:10+00:00  Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() at /var/www/owncloud/lib/composer/symfony/event-dispatcher/EventDispatcher.php:43
2018-01-12T18:20:10+00:00  Symfony\Component\EventDispatcher\EventDispatcher->dispatch() at /var/www/owncloud/lib/private/Repair/Apps.php:222
2018-01-12T18:20:10+00:00  OC\Repair\Apps->getAppsFromMarket() at /var/www/owncloud/lib/private/Repair/Apps.php:158
2018-01-12T18:20:10+00:00  OC\Repair\Apps->run() at /var/www/owncloud/lib/private/Repair.php:89
2018-01-12T18:20:10+00:00  OC\Repair->run() at /var/www/owncloud/lib/private/Updater.php:221
2018-01-12T18:20:10+00:00  OC\Updater->doUpgrade() at /var/www/owncloud/lib/private/Updater.php:108
2018-01-12T18:20:10+00:00  OC\Updater->upgrade() at /var/www/owncloud/core/Command/Upgrade.php:254
2018-01-12T18:20:10+00:00  OC\Core\Command\Upgrade->execute() at /var/www/owncloud/lib/composer/symfony/console/Command/Command.php:262
2018-01-12T18:20:10+00:00  Symfony\Component\Console\Command\Command->run() at /var/www/owncloud/lib/composer/symfony/console/Application.php:826
2018-01-12T18:20:10+00:00  Symfony\Component\Console\Application->doRunCommand() at /var/www/owncloud/lib/composer/symfony/console/Application.php:189
2018-01-12T18:20:10+00:00  Symfony\Component\Console\Application->doRun() at /var/www/owncloud/lib/composer/symfony/console/Application.php:120
2018-01-12T18:20:10+00:00  Symfony\Component\Console\Application->run() at /var/www/owncloud/lib/private/Console/Application.php:161
2018-01-12T18:20:10+00:00  OC\Console\Application->run() at /var/www/owncloud/console.php:106
2018-01-12T18:20:10+00:00  require_once() at /var/www/owncloud/occ:11
2018-01-12T18:20:10+00:00 
2018-01-12T18:20:10+00:00 upgrade [--no-app-disable]
2018-01-12T18:20:10+00:00 
```

**I tried it again, and output was slightly different:**

```
root@owncloud:/var/www/owncloud# sudo -u www-data php ./occ upgrade
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
2018-01-12T18:20:14+00:00 Set log level to debug
2018-01-12T18:20:14+00:00 Repair step: Repair MySQL database engine
2018-01-12T18:20:14+00:00 Repair step: Repair MySQL collation
2018-01-12T18:20:14+00:00 Repair info: All tables already have the correct collation -> nothing to do
2018-01-12T18:20:14+00:00 Repair step: Repair SQLite autoincrement
2018-01-12T18:20:14+00:00 Repair step: Repair duplicate entries in oc_lucene_status
2018-01-12T18:20:14+00:00 Repair info: lucene_status table does not exist -> nothing to do
2018-01-12T18:20:14+00:00 Repair step: Upgrade app code from the marketplace
2018-01-12T18:20:14+00:00 Repair info: Using market to update existing apps
2018-01-12T18:20:14+00:00 Repair info: Attempting to update the following missing apps from market: gallery, oauth2, xmas
2018-01-12T18:20:14+00:00 Repair info: Fetching app from market: gallery
2018-01-12T18:20:16+00:00 
2018-01-12T18:20:16+00:00                                                                                                                                                                                                     
2018-01-12T18:20:16+00:00   [Symfony\Component\Debug\Exception\FatalThrowableError]                                                                                                                                           
2018-01-12T18:20:16+00:00   Type error: Argument 2 passed to OC_App::executeRepairSteps() must be of the type array, null given, called in /var/www/owncloud/lib/private/Installer.php on line 154  
2018-01-12T18:20:16+00:00                                                                                                                                                                                                     

2018-01-12T18:20:16+00:00 Exception trace:
2018-01-12T18:20:16+00:00  () at /var/www/owncloud/lib/private/legacy/app.php:1064
2018-01-12T18:20:16+00:00  OC_App::executeRepairSteps() at /var/www/owncloud/lib/private/Installer.php:154
2018-01-12T18:20:16+00:00  OC\Installer::installApp() at /var/www/owncloud/lib/private/App/AppManager.php:436
2018-01-12T18:20:16+00:00  OC\App\AppManager->installApp() at /var/www/owncloud/apps/market/lib/MarketService.php:131
2018-01-12T18:20:16+00:00  OCA\Market\MarketService->installPackage() at /var/www/owncloud/apps/market/lib/MarketService.php:120
2018-01-12T18:20:16+00:00  OCA\Market\MarketService->installApp() at /var/www/owncloud/apps/market/lib/Listener.php:46
2018-01-12T18:20:16+00:00  OCA\Market\Listener->reinstallAppStoreApp() at /var/www/owncloud/apps/market/lib/Application.php:55
2018-01-12T18:20:16+00:00  OCA\Market\Application->OCA\Market\{closure}() at n/a:n/a
2018-01-12T18:20:16+00:00  call_user_func() at /var/www/owncloud/lib/composer/symfony/event-dispatcher/EventDispatcher.php:174
2018-01-12T18:20:16+00:00  Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() at /var/www/owncloud/lib/composer/symfony/event-dispatcher/EventDispatcher.php:43
2018-01-12T18:20:16+00:00  Symfony\Component\EventDispatcher\EventDispatcher->dispatch() at /var/www/owncloud/lib/private/Repair/Apps.php:222
2018-01-12T18:20:16+00:00  OC\Repair\Apps->getAppsFromMarket() at /var/www/owncloud/lib/private/Repair/Apps.php:158
2018-01-12T18:20:16+00:00  OC\Repair\Apps->run() at /var/www/owncloud/lib/private/Repair.php:89
2018-01-12T18:20:16+00:00  OC\Repair->run() at /var/www/owncloud/lib/private/Updater.php:221
2018-01-12T18:20:16+00:00  OC\Updater->doUpgrade() at /var/www/owncloud/lib/private/Updater.php:108
2018-01-12T18:20:16+00:00  OC\Updater->upgrade() at /var/www/owncloud/core/Command/Upgrade.php:254
2018-01-12T18:20:16+00:00  OC\Core\Command\Upgrade->execute() at /var/www/owncloud/lib/composer/symfony/console/Command/Command.php:262
2018-01-12T18:20:16+00:00  Symfony\Component\Console\Command\Command->run() at /var/www/owncloud/lib/composer/symfony/console/Application.php:826
2018-01-12T18:20:16+00:00  Symfony\Component\Console\Application->doRunCommand() at /var/www/owncloud/lib/composer/symfony/console/Application.php:189
2018-01-12T18:20:16+00:00  Symfony\Component\Console\Application->doRun() at /var/www/owncloud/lib/composer/symfony/console/Application.php:120
2018-01-12T18:20:16+00:00  Symfony\Component\Console\Application->run() at /var/www/owncloud/lib/private/Console/Application.php:161
2018-01-12T18:20:16+00:00  OC\Console\Application->run() at /var/www/owncloud/console.php:106
2018-01-12T18:20:16+00:00  require_once() at /var/www/owncloud/occ:11
2018-01-12T18:20:16+00:00 
2018-01-12T18:20:16+00:00 upgrade [--no-app-disable]
2018-01-12T18:20:16+00:00 
```

**I tried it again, and output was slightly different:**

```
root@owncloud:/var/www/owncloud# sudo -u www-data php ./occ upgrade
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
2018-01-12T18:20:22+00:00 Set log level to debug
2018-01-12T18:20:22+00:00 Repair step: Repair MySQL database engine
2018-01-12T18:20:22+00:00 Repair step: Repair MySQL collation
2018-01-12T18:20:22+00:00 Repair info: All tables already have the correct collation -> nothing to do
2018-01-12T18:20:22+00:00 Repair step: Repair SQLite autoincrement
2018-01-12T18:20:22+00:00 Repair step: Repair duplicate entries in oc_lucene_status
2018-01-12T18:20:22+00:00 Repair info: lucene_status table does not exist -> nothing to do
2018-01-12T18:20:22+00:00 Repair step: Upgrade app code from the marketplace
2018-01-12T18:20:22+00:00 Repair info: Using market to update existing apps
2018-01-12T18:20:22+00:00 Repair info: Attempting to update the following missing apps from market: oauth2, xmas
2018-01-12T18:20:22+00:00 Repair info: Fetching app from market: oauth2
2018-01-12T18:20:23+00:00 Repair warning: Exception
2018-01-12T18:20:23+00:00 Repair warning: App not found
2018-01-12T18:20:23+00:00 Repair info: Fetching app from market: xmas
2018-01-12T18:20:25+00:00 
2018-01-12T18:20:25+00:00                                                                                                                                                                                                     
2018-01-12T18:20:25+00:00   [Symfony\Component\Debug\Exception\FatalThrowableError]                                                                                                                                           
2018-01-12T18:20:25+00:00   Type error: Argument 2 passed to OC_App::executeRepairSteps() must be of the type array, null given, called in /var/www/owncloud/lib/private/Installer.php on line 154  
2018-01-12T18:20:25+00:00                                                                                                                                                                                                     

2018-01-12T18:20:25+00:00 Exception trace:
2018-01-12T18:20:25+00:00  () at /var/www/owncloud/lib/private/legacy/app.php:1064
2018-01-12T18:20:25+00:00  OC_App::executeRepairSteps() at /var/www/owncloud/lib/private/Installer.php:154
2018-01-12T18:20:25+00:00  OC\Installer::installApp() at /var/www/owncloud/lib/private/App/AppManager.php:436
2018-01-12T18:20:25+00:00  OC\App\AppManager->installApp() at /var/www/owncloud/apps/market/lib/MarketService.php:131
2018-01-12T18:20:25+00:00  OCA\Market\MarketService->installPackage() at /var/www/owncloud/apps/market/lib/MarketService.php:120
2018-01-12T18:20:25+00:00  OCA\Market\MarketService->installApp() at /var/www/owncloud/apps/market/lib/Listener.php:46
2018-01-12T18:20:25+00:00  OCA\Market\Listener->reinstallAppStoreApp() at /var/www/owncloud/apps/market/lib/Application.php:55
2018-01-12T18:20:25+00:00  OCA\Market\Application->OCA\Market\{closure}() at n/a:n/a
2018-01-12T18:20:25+00:00  call_user_func() at /var/www/owncloud/lib/composer/symfony/event-dispatcher/EventDispatcher.php:174
2018-01-12T18:20:25+00:00  Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() at /var/www/owncloud/lib/composer/symfony/event-dispatcher/EventDispatcher.php:43
2018-01-12T18:20:25+00:00  Symfony\Component\EventDispatcher\EventDispatcher->dispatch() at /var/www/owncloud/lib/private/Repair/Apps.php:222
2018-01-12T18:20:25+00:00  OC\Repair\Apps->getAppsFromMarket() at /var/www/owncloud/lib/private/Repair/Apps.php:158
2018-01-12T18:20:25+00:00  OC\Repair\Apps->run() at /var/www/owncloud/lib/private/Repair.php:89
2018-01-12T18:20:25+00:00  OC\Repair->run() at /var/www/owncloud/lib/private/Updater.php:221
2018-01-12T18:20:25+00:00  OC\Updater->doUpgrade() at /var/www/owncloud/lib/private/Updater.php:108
2018-01-12T18:20:25+00:00  OC\Updater->upgrade() at /var/www/owncloud/core/Command/Upgrade.php:254
2018-01-12T18:20:25+00:00  OC\Core\Command\Upgrade->execute() at /var/www/owncloud/lib/composer/symfony/console/Command/Command.php:262
2018-01-12T18:20:25+00:00  Symfony\Component\Console\Command\Command->run() at /var/www/owncloud/lib/composer/symfony/console/Application.php:826
2018-01-12T18:20:25+00:00  Symfony\Component\Console\Application->doRunCommand() at /var/www/owncloud/lib/composer/symfony/console/Application.php:189
2018-01-12T18:20:25+00:00  Symfony\Component\Console\Application->doRun() at /var/www/owncloud/lib/composer/symfony/console/Application.php:120
2018-01-12T18:20:25+00:00  Symfony\Component\Console\Application->run() at /var/www/owncloud/lib/private/Console/Application.php:161
2018-01-12T18:20:25+00:00  OC\Console\Application->run() at /var/www/owncloud/console.php:106
2018-01-12T18:20:25+00:00  require_once() at /var/www/owncloud/occ:11
2018-01-12T18:20:25+00:00 
2018-01-12T18:20:25+00:00 upgrade [--no-app-disable]
2018-01-12T18:20:25+00:00 
```

**I tried it again, and output looked much better:**

```
root@owncloud:/var/www/owncloud# sudo -u www-data php ./occ upgrade
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
2018-01-12T18:22:03+00:00 Set log level to debug
2018-01-12T18:22:03+00:00 Repair step: Repair MySQL database engine
2018-01-12T18:22:03+00:00 Repair step: Repair MySQL collation
2018-01-12T18:22:03+00:00 Repair info: All tables already have the correct collation -> nothing to do
2018-01-12T18:22:03+00:00 Repair step: Repair SQLite autoincrement
2018-01-12T18:22:03+00:00 Repair step: Repair duplicate entries in oc_lucene_status
2018-01-12T18:22:03+00:00 Repair info: lucene_status table does not exist -> nothing to do
2018-01-12T18:22:03+00:00 Repair step: Upgrade app code from the marketplace
2018-01-12T18:22:03+00:00 Repair info: Using market to update existing apps
2018-01-12T18:22:03+00:00 Repair info: Attempting to update the following existing compatible apps from market: 
2018-01-12T18:22:03+00:00 Repair info: Fetching app from market: activity
2018-01-12T18:22:11+00:00 Repair info: 
2018-01-12T18:22:11+00:00 Repair info: Fetching app from market: announcementcenter
2018-01-12T18:22:11+00:00 Repair info: 
2018-01-12T18:22:11+00:00 Repair info: Fetching app from market: comments
2018-01-12T18:22:11+00:00 Repair info: App (comments) is not known at the marketplace.
2018-01-12T18:22:11+00:00 Repair info: Fetching app from market: configreport
2018-01-12T18:22:11+00:00 Repair info: App (configreport) is not known at the marketplace.
2018-01-12T18:22:11+00:00 Repair info: Fetching app from market: dav
2018-01-12T18:22:12+00:00 Repair info: App (dav) is not known at the marketplace.
2018-01-12T18:22:12+00:00 Repair info: Fetching app from market: federatedfilesharing
2018-01-12T18:22:12+00:00 Repair info: App (federatedfilesharing) is not known at the marketplace.
2018-01-12T18:22:12+00:00 Repair info: Fetching app from market: federation
2018-01-12T18:22:12+00:00 Repair info: App (federation) is not known at the marketplace.
2018-01-12T18:22:12+00:00 Repair info: Fetching app from market: files
2018-01-12T18:22:13+00:00 Repair info: App (files) is not known at the marketplace.
2018-01-12T18:22:13+00:00 Repair info: Fetching app from market: files_external
2018-01-12T18:22:13+00:00 Repair info: App (files_external) is not known at the marketplace.
2018-01-12T18:22:13+00:00 Repair info: Fetching app from market: files_pdfviewer
2018-01-12T18:22:13+00:00 Repair info: 
2018-01-12T18:22:13+00:00 Repair info: Fetching app from market: files_sharing
2018-01-12T18:22:14+00:00 Repair info: App (files_sharing) is not known at the marketplace.
2018-01-12T18:22:14+00:00 Repair info: Fetching app from market: files_texteditor
2018-01-12T18:22:14+00:00 Repair info: 
2018-01-12T18:22:14+00:00 Repair info: Fetching app from market: files_trashbin
2018-01-12T18:22:14+00:00 Repair info: App (files_trashbin) is not known at the marketplace.
2018-01-12T18:22:14+00:00 Repair info: Fetching app from market: files_versions
2018-01-12T18:22:15+00:00 Repair info: App (files_versions) is not known at the marketplace.
2018-01-12T18:22:15+00:00 Repair info: Fetching app from market: files_videoplayer
2018-01-12T18:22:15+00:00 Repair info: App (files_videoplayer) is not known at the marketplace.
2018-01-12T18:22:15+00:00 Repair info: Fetching app from market: firstrunwizard
2018-01-12T18:22:15+00:00 Repair info: App (firstrunwizard) is not known at the marketplace.
2018-01-12T18:22:15+00:00 Repair info: Fetching app from market: gallery
2018-01-12T18:22:15+00:00 Repair info: 
2018-01-12T18:22:15+00:00 Repair info: Fetching app from market: market
2018-01-12T18:22:16+00:00 Repair info: 
2018-01-12T18:22:16+00:00 Repair info: Fetching app from market: notifications
2018-01-12T18:22:16+00:00 Repair info: App (notifications) is not known at the marketplace.
2018-01-12T18:22:16+00:00 Repair info: Fetching app from market: oauth2
2018-01-12T18:22:16+00:00 Repair info: 
2018-01-12T18:22:16+00:00 Repair info: Fetching app from market: provisioning_api
2018-01-12T18:22:17+00:00 Repair info: App (provisioning_api) is not known at the marketplace.
2018-01-12T18:22:17+00:00 Repair info: Fetching app from market: templateeditor
2018-01-12T18:22:17+00:00 Repair info: App (templateeditor) is not known at the marketplace.
2018-01-12T18:22:17+00:00 Repair info: Fetching app from market: updatenotification
2018-01-12T18:22:17+00:00 Repair info: App (updatenotification) is not known at the marketplace.
2018-01-12T18:22:17+00:00 Repair info: Fetching app from market: xmas
2018-01-12T18:22:18+00:00 Repair info: 
2018-01-12T18:22:18+00:00 Repair info: App was not updated: activity
2018-01-12T18:22:18+00:00 Repair info: App was not updated: announcementcenter
2018-01-12T18:22:18+00:00 Repair info: App was not updated: comments
2018-01-12T18:22:18+00:00 Repair info: App was not updated: configreport
2018-01-12T18:22:18+00:00 Repair info: App was not updated: dav
2018-01-12T18:22:18+00:00 Repair info: App was not updated: federatedfilesharing
2018-01-12T18:22:18+00:00 Repair info: App was not updated: federation
2018-01-12T18:22:18+00:00 Repair info: App was not updated: files
2018-01-12T18:22:18+00:00 Repair info: App was not updated: files_external
2018-01-12T18:22:18+00:00 Repair info: App was not updated: files_pdfviewer
2018-01-12T18:22:18+00:00 Repair info: App was not updated: files_sharing
2018-01-12T18:22:18+00:00 Repair info: App was not updated: files_texteditor
2018-01-12T18:22:18+00:00 Repair info: App was not updated: files_trashbin
2018-01-12T18:22:18+00:00 Repair info: App was not updated: files_versions
2018-01-12T18:22:18+00:00 Repair info: App was not updated: files_videoplayer
2018-01-12T18:22:18+00:00 Repair info: App was not updated: firstrunwizard
2018-01-12T18:22:18+00:00 Repair info: App was not updated: gallery
2018-01-12T18:22:18+00:00 Repair info: App was not updated: market
2018-01-12T18:22:18+00:00 Repair info: App was not updated: notifications
2018-01-12T18:22:18+00:00 Repair info: App was not updated: oauth2
2018-01-12T18:22:18+00:00 Repair info: App was not updated: provisioning_api
2018-01-12T18:22:18+00:00 Repair info: App was not updated: templateeditor
2018-01-12T18:22:18+00:00 Repair info: App was not updated: updatenotification
2018-01-12T18:22:18+00:00 Repair info: App was not updated: xmas
2018-01-12T18:22:18+00:00 Updating database schema
2018-01-12T18:22:18+00:00 Updated database
2018-01-12T18:22:18+00:00 Updating <templateeditor> ...
2018-01-12T18:22:18+00:00 Updated <templateeditor> to 0.2
2018-01-12T18:22:18+00:00 Repair step: Repair mime types
2018-01-12T18:22:18+00:00 Repair step: Detect file cache entries with path that does not match parent-child relationships
2018-01-12T18:22:18+00:00 Repair step: Generate ETags for file where no ETag is present.
2018-01-12T18:22:18+00:00 Repair info: ETags have been fixed for 0 files/folders.
2018-01-12T18:22:18+00:00 Repair step: Clean tags and favorites
2018-01-12T18:22:18+00:00 Repair info: 0 tags of deleted users have been removed.
2018-01-12T18:22:18+00:00 Repair info: 0 tags for delete files have been removed.
2018-01-12T18:22:18+00:00 Repair info: 0 tag entries for deleted tags have been removed.
2018-01-12T18:22:18+00:00 Repair info: 0 tags with no entries have been removed.
2018-01-12T18:22:18+00:00 Repair step: Drop old database tables
2018-01-12T18:22:18+00:00 Drop old database tables
2018-01-12T18:22:18+00:00 
                                                    2018-01-12T18:22:18+00:00  Done
 28/28 [============================] 100%2018-01-12T18:22:18+00:00 
2018-01-12T18:22:18+00:00 Repair step: Drop old background jobs
2018-01-12T18:22:18+00:00 Repair step: Remove getetag entries in properties table
2018-01-12T18:22:18+00:00 Repair info: Removed 0 unneeded "{DAV:}getetag" entries from properties table.
2018-01-12T18:22:18+00:00 Repair step: Repair outdated OCS IDs
2018-01-12T18:22:18+00:00 Repair step: Repair invalid shares
2018-01-12T18:22:18+00:00 Repair step: Remove old share propagation app entries
2018-01-12T18:22:18+00:00 Repair step: Move user avatars outside the homes to the new location
2018-01-12T18:22:18+00:00 Repair step: Remove shares of a users root folder
2018-01-12T18:22:18+00:00 Repair step: Repair unmerged shares
2018-01-12T18:22:18+00:00 Repair step: Disable extra themes
2018-01-12T18:22:18+00:00 Starting code integrity check...
2018-01-12T18:22:30+00:00 Finished code integrity check
2018-01-12T18:22:30+00:00 Update successful
2018-01-12T18:22:30+00:00 Maintenance mode is kept active
2018-01-12T18:22:30+00:00 Reset log level
```

**I performed a quick check:**

```
root@owncloud:/var/www/owncloud# sudo -u www-data php ./occ app:list
Enabled:
  - activity: 2.3.6
  - announcementcenter: 1.2.0
  - comments: 0.3.0
  - configreport: 0.1.1
  - dav: 0.3.2
  - federatedfilesharing: 0.3.1
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.7.1
  - files_pdfviewer: 0.8.2
  - files_sharing: 0.10.1
  - files_texteditor: 2.2.1
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 16.0.2
  - market: 0.2.3
  - notifications: 0.3.2
  - oauth2: 0.2.1
  - provisioning_api: 0.5.0
  - templateeditor: 0.2
  - updatenotification: 0.2.1
  - xmas: 0.0.4
Disabled:
  - encryption
  - external
  - systemtags
  - theme-example
  - user_external
```

### Server configuration
**Operating system**:
Ubuntu Xenial (16.04 LTS) (GNU/Linux 4.5.7-std-3 x86_64 )

**Web server:**
Apache/2.4.18 (Ubuntu)

**Database:**
mysql  Ver 14.14 Distrib 5.7.20, for Linux (x86_64) using  EditLine wrapper

**PHP version:**
"PHP Version": "7.0.22-0ubuntu0.16.04.1",

**ownCloud version:** (see ownCloud admin page)
"ownCloud version string": "10.0.5 RC3",

**Updated from an older ownCloud or fresh install:**
Updated from 10.0.4

**Where did you install ownCloud from:**
*.tar

**Signing status (ownCloud 9.0 and above):**

```
No errors have been found.
```

**The content of config/config.php:**

```
Log in to the web-UI with an administrator account and click on
'admin' -> 'Generate Config Report' -> 'Download ownCloud config report'
This report includes the config.php settings, the list of activated apps
and other details in a well sanitized form.

or 

If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

*ATTENTION:* Do not post your config.php file in public as is. Please use one of the above
methods whenever possible. Both, the generated reports from the web-ui and from occ config:list
consistently remove sensitive data. You still may want to review the report before sending.
If done manually then it is critical for your own privacy to dilligently
remove *all* host names, passwords, usernames, salts and other credentials before posting.
You should assume that attackers find such information and will use them against your systems.
```

**List of activated apps:**

```
root@owncloud:/var/www/owncloud# sudo -u www-data php ./occ app:list
Enabled:
  - activity: 2.3.6
  - announcementcenter: 1.2.0
  - comments: 0.3.0
  - configreport: 0.1.1
  - dav: 0.3.2
  - federatedfilesharing: 0.3.1
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.7.1
  - files_pdfviewer: 0.8.2
  - files_sharing: 0.10.1
  - files_texteditor: 2.2.1
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 16.0.2
  - market: 0.2.3
  - notifications: 0.3.2
  - oauth2: 0.2.1
  - provisioning_api: 0.5.0
  - templateeditor: 0.2
  - updatenotification: 0.2.1
  - xmas: 0.0.4
Disabled:
  - encryption
  - external
  - systemtags
  - theme-example
  - user_external
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

**Comment by VicDeo** [comment]
it's the same with https://github.com/owncloud/core/issues/30112

**Comment by VicDeo** [comment]
@michaelstingl the fix has been merged into 10.0.5.rc4
